### PR TITLE
RCPS-66 desktop-frontend-maintenance: Refactor Login Module

### PR DESCRIPTION
**Pull request by Conconrad**
ACCEPTANCE CRITERIA:

The code should be refactored based on the coding standards.

All unnecessary codes must be removed.

The system’s functionality must be working even after refactoring.

There should be no bugs brought by the refactoring.

### CODE REVIEW

**Review by zhairanicolepacheco — CHANGES_REQUESTED** [review]
.idea folder is included in the commit, please add .idea/ to the .gitignore file to prevent it from being tracked

**Review by zhairanicolepacheco — CHANGES_REQUESTED** [review]
do not commit the modal component as we refactor changes per function, comment out or remove it's usage in the login module

File: porms-ums-application/src/main/java/com/system/demo/appl/facade/user/impl/UserFacadeImpl.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+/**
+ * An implementation class of the User Facade.
+ */
+
+public class UserFacadeImpl implements UserFacade {
+
+    /**
+     * The logger for this class.
```
Comment (esdi-00) [inline porms-ums-application/src/main/java/com/system/demo/appl/facade/user/impl/UserFacadeImpl.java:16]: logger?

File: porms-ums-application/src/main/java/com/system/demo/UserInfoMgtApplication.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package com.system.demo;
+
+import com.system.demo.appl.facade.user.UserFacade;
+import com.system.demo.appl.facade.user.impl.UserFacadeImpl;
+import com.system.demo.data.user.dao.UserDao;
+import com.system.demo.data.user.dao.impl.UserDaoImpl;
+
+public class UserInfoMgtApplication {
```
Comment (esdi-00) [inline porms-ums-application/src/main/java/com/system/demo/UserInfoMgtApplication.java:8]: missing javadocs

File: porms-ums-application/src/main/java/com/system/demo/data/connection/ConnectionHelper.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+public class ConnectionHelper {
+    public static String URL = "jdbc:oracle:thin:@localhost:1521:rogate";
+    /**
+     * The Oracle driver.
+     */
+    public static String ORACLE_DRIVER = "oracle.jdbc.driver.OracleDriver";
+    /**
+     * The username used to connect to the database.
+     */
+    public static String username = "system";
```
Comment (esdi-00) [inline porms-ums-application/src/main/java/com/system/demo/data/connection/ConnectionHelper.java:19]: incorrect user

**Review by esdi-00 — CHANGES_REQUESTED** [review]
provide all the necessary documentations
check other comments

**Review by esdi-00 — CHANGES_REQUESTED** [review]
why is the repo inside another folder porms-ums-application?
why check in .idea codes?
still missing some javadocs

**Pull request closed without merging**